util/httputil/http_util.go

```diff
@@ -52,7 +52,7 @@ func HttpPost(url string, auth *config.Auth, body io.Reader, timeout time.Durati
 		return nil, 0, err
 	}
 
-	transport, err := CreateTransport(auth, &http.Transport{}, timeout)
+	transport, err := CreateTransport(auth, &http.Transport{}, timeout, map[string]string{})
```

Comment:
pass in nil - that's how every other place is that I changed that added this new argument.